controllers/apply/under10k/form.test.js

```diff
@@ -687,20 +687,18 @@ test('finance details required if organisation is over 15 months old', function
 
     const validData = mockResponse({
         organisationStartDate: { month: now.month() + 1, year: now.year() },
-        accountingYearDate: null,
```

Comment:
This is the one change I'm not sure why we needed to make it. Actual form looked OK, so think this is more a difference in null vs nonexistent properties when we are testing the form like this. Need to double check this one.